### ISSUE TITLE
Video Picture in Picture (PiP) is too small

### ISSUE THREAD

**Issue by 360fun** [TeamNewPipe/NewPipe#4375]
In my Android tablet the PiP is very very small and I found no option to change the size...am I missing something or this is normal?

![PIP](https://user-images.githubusercontent.com/844090/95000080-06be6c00-05be-11eb-96a9-95f506fec49f.jpg)

**Comment by Stypox** [comment]
You can resize it at your will with the expected two-finger gestures, or you can long press on it to make it as large as possible

**Comment by 360fun** [comment]
> You can resize it at your will with the expected two-finger gestures, or you can long press on it to make it as large as possible

I tried but I could not manage to do it the first time, now I tried again being more careful and I did it...not very intuitive.

I try to explain the problem:
I confirm that works but it's a bit buggy and doesn't feel the fingers if you're fast. While I enlarge the PiP the size does not change but it moves away from the fingers (goes up left), after a bit it reaches the correct size. It's like the virtual PiP object enlarges but the UI syncs with a delay.
In fact if I zoom veeery slow, it's kind of ok, just is not smooth and very shaky/flickery. 🤔
